ifme/lang/tr.ini

```diff
@@ -85,10 +85,10 @@ SelectOneVideoAttch = Lütfen ekleri kaldırmadan önce bir video seçin!
 SelectOneVideoPreview = Lütfen önizleme için tek video seçin!
 SelectOneVideoSubtitle = Lütfen altyazıları kaldırmadan önce bir video seçin!
 VideoToAviSynth = Seçili dosyayı AviSynth komutuna çevirmek istiyor musunuz?\n(FFmpeg çözücüsünü esgeçmek için kullanılır. Desteklenmeyen dosya atlanır.)
-QueueSave = Bu sıra kaydetmek istiyor musunuz? Bu CLI üzerinden kitlesel kodlama için niyetinde.
-QueueSaveError = En az 2 ya da daha fazla kuyruk
-QueueOpenChange = Untitled değişiklikleri kaydetmek istiyor musunuz?
-Quit = Bu kuyruk listesinden çıkmak ve kaydetmek istiyor musunuz?
+QueueSave = Bu kuyruğu kaydetmek istiyor musunuz? Daha sonra tekrar açabilir veya komut satırından çalıştırabilirsiniz.
+QueueSaveError = Kuyruktan en az 2 öğe seçmelisiniz.
```

Comment:
If meaning = "You must pick at least two item before you save" Then
   leave as is
ElseIf meaning = "There must be at least two files to save queue" Then
   edit = "Kaydedebilmek için kuyrukta en az 2 öğe olmalıdır."
Else
   I dunno, tell me about it.
End If